### ISSUE TITLE
Website structure

### ISSUE THREAD

**Issue by blackgear** [suisei-cn/code-of-conduct#1]
- Portal: main entry
- Voice: starbutton
- Music: music list with a player
- Gallery: fanarts collections
- Tetris: a playable tetris
- Community: a introduction pages for official or unofficial commnunity

**Comment by outloudvi** [comment]
Okay, so this is our first RFC.

1. I think what we are really discussing here might be the design of the portal. If so, the issue title can be more accurate such as "Portal design".
2. In your RFC, will the portal replace starbuttons as the content of <https://suisei.moe/index.html>?

**Comment by George-Miao** [comment]
`suisei.moe` can be served as an overall entry (portal) of the site. And the structure of the site should be unified. `player.suisei.moe` and `suisei.moe/podcast.xml` are conflicted in regard to structure. Either put everything under `suisei.moe` or spread them like `subdomain.suisei.moe`. For instance, starbutton can be hosted on `button.suisei.moe` or `suisei.moe/button` etc. 

Here is an example of structure: 

```
.    suisei.moe - Portal
| -- button.suisei.moe - Starbutton
| -- music.suisei.moe - Music web player
|    ` -- music.suisei.moe/podcast.xml - Podcast sub
` -- community.suisei.moe - Introduction
```

**Comment by outloudvi** [comment]
I agree that it would be great to unify the structure. However, there are several problems to consider on this approach:

1. Move `suisei.moe/podcast.xml`: It's simple and just requires configuration on the Cloudflare side. However, for compatibility, it would need a redirect from the old site. Since Cloudflare has a limit on the number of workers, we can let them use the same one worker and use hostnames to tell between them.
2. On the introduction, I would say there have been enough introduction websites for Suisei (the old [k-k.com](https://kyoumo-kawaii.com/), also the recent [h-o-s.w.i](https://hoshiyomi-observatory-suichankawaii.webflow.io/)). And actually, we also had a project like that, which is [suisei-cn/hosimati.suisei.moe](https://github.com/suisei-cn/hosimati.suisei.moe), also deprecated somehow. It's obviously a content-centric project, and can only proceed if we find people who are willing to contribute content on that.
3. Also, the design on the overall entry. blackgear's idea seems intriguing to me, but detailed discussion on aesthetics would be preferred.
